pipeline/06-upload.R

```diff
@@ -0,0 +1,320 @@
+#- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+# 1. Setup ---------------------------------------------------------------------
+#- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
```

Comment:
The code in this file was extracted from `05-finalize.R` with very few changes. Since they aren't visible in the diff, I'll call them out in comments below.